src/actions/index.js

```diff
@@ -19,9 +19,12 @@ export const setWallet = payload => ({
 });
 
 export const selectCoin = (selectedCoins, pairs) => dispatch => {
-  let validPair = pairs && Object.keys(pairs[selectedCoins.deposit]).includes(selectedCoins.receive);
+  let pairIsValid = true;
```

Comment:
@punit2502 what is the point of having this variable if it is just hard coded as true

Reply:
its true only initally

if pair is changed by user and and pair is invalid then its value changes - https://github.com/onitsoft/nexchange-open-client-react/pull/733/files#diff-4e3037116dbc2bc69a473b7ace3d02abR24-R25